deploy/crds/mattermost_v1alpha1_clusterinstallation_crd.yaml

```diff
@@ -39,6 +39,10 @@ spec:
               type: object
             databaseType:
               properties:
+                externalDatabase:
+                  description: If the user want to use an external DB Can be inside
```

Comment:
nit: `external DB Can be` should maybe be `external DB. This can be...`?

Reply:
done